modules/ROOT/pages/proc_checking-pipeline-and-application-status.adoc

```diff
@@ -14,23 +14,18 @@ You can examine the logs for `PipelineRuns` to verify its status.
 +
 ----
 $ tkn pipeline logs -f
-? Select pipeline : petclinic-deploy-pipeline
-? Select pipelinerun : petclinic-deploy-pipeline-run-tsv92 started 39 seconds ago
-
-...
-[build : nop] Build successful
-[deploy : build-step-oc] deploymentconfig.apps.openshift.io/spring-petclinic rolled out
-[deploy : nop] Build successful
+? Select pipeline : build-and-deploy
+? Select pipelinerun : build-and-deploy-run-tsv92 started 39 seconds ago
 ----
 
-. Check the *Project Status* in the OpenShift Web Console to verify that the PetClinic image is successfully built and deployed.
+. Check the *Project Status* in the OpenShift Web Console to verify that the image is successfully built and deployed.
```

Comment:
`that the image` -> `that the images`. In this flow pipeline going to build more than one image, right?

Reply:
Accepted. Will incorporate